vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/EventHandlingIT.java

```diff
@@ -217,6 +219,24 @@ public void invalidFieldsIndicatedOnSave() {
         Assert.assertTrue(lastNameField.hasAttribute("invalid"));
     }
 
+    @Test
+    public void newEventItem() {
```

Comment:
Naming: It's not clear that this tests that the new item can be initialized / prepared in the new event listener.